src/components/auth/Auth.js

```diff
@@ -19,21 +16,12 @@ function Auth({ children }) {
 
       authService.firebase.auth().onAuthStateChanged((user) => {
         dispatch(setUserData(user));
-        if (user) {
-          user.getIdToken().then((token) => {
-            authService.setSession(token);
-          });
-        }
```

Comment:
Firebase access token expires every one hour, so if you don't do this the token will persist when you first logged in. That same token won't work after an hour. So needed to change that token after every state change.

Reply:
> Firebase access token expires every one hour, so if you don't do this the token will persist when you first logged in. That same token won't work after an hour. So needed to change that token after every state change.

OOh Okay !! sure  @Abhishek-kumar09  I'll revert this change  :)